Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -6577,6 +6577,7 @@ bool CStaticFunctionDefinitions::SetVehicleWheelStates(CElement* pElement, int i
 
                 CBitStream BitStream;
                 BitStream.pBitStream->Write((const char*)&pVehicle->m_ucWheelStates[0], MAX_WHEELS);
+                BitStream.pBitStream->WriteBit(spawnFlyingComponent);
```

Comment:
I think you need a `BitStream.Can` here as well.

Reply:
setVehicleDoorState() does not use it so I guess not. The only way that it'll read the bit is if it's on the same version and that only depends on the server.